draft-thomson-http-mice.md

```diff
@@ -336,19 +326,19 @@ representation.
 ~~~
 PUT /test HTTP/1.1
 Host: example.com
-MI: mi-sha256=IVa9shfs0nyKEhHqtB3WVNANJ2Njm5KjQLjRtnbkYJ4
+Digest: mi-sha256=IVa9shfs0nyKEhHqtB3WVNANJ2Njm5KjQLjRtnbkYJ4=
 Content-Encoding: mi-sha256
 Content-Length: 113
 
 <0x0000000000000010>When I grow up,
-OElbplJlPK-Rv6JNK6p5_515IaoPoZo-2elWL7OQ60A
+OElbplJlPK+Rv6JNK6p5/515IaoPoZo+2elWL7OQ60A=
 I want to be a w
-iPMpmgExHPrbEX3_RvwP4d16fWlK4l--p75PUu_KyN0
+iPMpmgExHPrbEX3/RvwP4d16fWlK4l++p75PUu_KyN0=
 atermelon
 ~~~
 
 Since the inline integrity proofs contain non-printing characters, these are
-shown here using the base64url encoding [RFC7515] with new lines between the
+shown here using the base64 encoding [RFC4648] with new lines between the
```

Comment:
You can save effort by using `{{!RFC4648}}` here.  I didn't with this doc originally because it predates that capability in the tools.

Reply:
Ah, ok. I had that originally and then matched your existing style. :)